android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -712,87 +712,6 @@ public boolean accept(File pathname) {
     }
   }
 
-  public static void updateOldKeyboardsList(Context context) {
-    ArrayList<HashMap<String, String>> kbList = KeyboardPickerActivity.getKeyboardsList(context);
-    if (kbList != null && kbList.size() > 0) {
```

Comment:
What happened to this function?  Why are we outright removing it?  Is it not needed for legacy / migration support?

Or does it reappear elsewhere, but greatly simplified?  If so, please link a reference to the new version for comparison.

Reply:
updateOldKeyboardsList was migrating `us` and `european2` keyboards which go back to Keyman for Android 2.8.
The Keyman app now installs sil_euro_latin so those old keyboards aren't needed.

Reply:
btw, migrating old keyboards will be re-addressed when I tackle keyboard updates
(previous "cloud" keyboards will be replaced when the keyboard package gets installed)

Reply:
To be sure we don't lose the reference and/or forget to implement this when the time comes, maybe make an issue regarding "old keyboard migration" for alpha.  It _will_ be an issue for anyone who decides to hop on board the current alpha, after all.  It can also keep a reference to this point in case the old code proves useful when writing the new migration code.

Reply:
k, to track as #3134